### PR TITLE
Update integration test and remove documents parameter from Spec2Vec

### PR DESCRIPTION
**Pull request by florian-huber**
Recent fixes in matchms correctly remove negative losses. That required a retrained word2vec model.
- New model file added.
- Removed ``documents`` argument from ``Spec2Vec`` and ``Spec2VecParallel`` since it is not needed.
- Switch from matchms 0.3.1 to matchms 0.3.4 since that contains the fixed ``add_losses`` filter.

### CODE REVIEW

File: spec2vec/Spec2Vec.py [hunk 1]
```diff
@@ -1,13 +1,11 @@
-import gensim
 import scipy
 from .calc_vector import calc_vector
 
 
 class Spec2Vec:
 
-    def __init__(self, model=None, documents=None, intensity_weighting_power=0):
+    def __init__(self, model=None, intensity_weighting_power=0):
```
Comment (sverhoeven) [inline spec2vec/Spec2Vec.py]: Model is required
```suggestion
    def __init__(self, model, intensity_weighting_power=0):
```

File: spec2vec/Spec2VecParallel.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
-import gensim
 import numpy
 import scipy
 from .calc_vector import calc_vector
 
 
 class Spec2VecParallel:
 
-    def __init__(self, model=None, documents=None, intensity_weighting_power=0):
+    def __init__(self, model=None, intensity_weighting_power=0):
```
Comment (sverhoeven) [inline spec2vec/Spec2VecParallel.py]: Model is required
```suggestion
    def __init__(self, model, intensity_weighting_power=0):
```

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Looks good, just some minor suggestions

File: spec2vec/Spec2Vec.py (hunk 1 above)
Comment (florian-huber) [reply spec2vec/Spec2Vec.py]: True, good point.

**Review by sverhoeven — APPROVED** [review]

**Comment by florian-huber** [comment]
Thanks for the review Stefan!

**Pull request merged**